packages/react-composites/src/composites/localization/locales/en-US/strings.json

```diff
@@ -36,6 +36,8 @@
     "mutedMessage": "You're muted",
     "networkReconnectMoreDetails": "Looks like something went wrong. We're trying to get back into the call.",
     "networkReconnectTitle": "Hold on",
+    "notInvitedToRoomDetails": "You were not invited to this room.",
```

Comment:
nit: normally we talk about not having permission to join instead of being 'invited' (e.g. you could be unauthorized for reasons other than an invitation (e.g. if you were invited but your invite subsequently revoked).
Just want to check we shouldn't go with something like `deniedPermissionToRoom`

Reply:
I'd reconsider the string values as well. The message "not invited" was a bit confusing to me in the UI snapshots.

"You do not have access" would have sounded more assertive.

But, if we can, let's get UX input for the string values.